docs/general/how-to-copy-the-contents-of-an-open-xml-package-part-to-a-document-part-in-a-dif.md

```diff
@@ -78,7 +78,7 @@ long as you use **using**.
 --------------------------------------------------------------------------------
 ## The Theme Part
 The theme part contains information about the color, font, and format of
-a document. It is defined in the [ISO/IEC 29500](https://www.iso.org/standard/71691.html) specification as
+a document. It is defined in the [!include[ISO/IEC 29500 URL](../includes/iso-iec-29500-link.md)] specification as
```

Comment:
```suggestion
a document. It is defined in the [!include[ISO/IEC 29500](../includes/iso-iec-29500-link.md)] specification as
```

does it need the "URL" as well?

Reply:
I did this to differentiate it from the version snippet. It does not show in the documentation, it's only the description for screen readers.

Reply:
oh that makes sense